### DISCUSSION TITLE
Exhaust Report issue

### DISCUSSION THREAD

**Discussion by shridhar95** [Sunbird-Lern/Community#123] [Issues] [answered]
Hi,

1.	When request for report is submitted the data gets store in which db and in which table?
2.	Some users can’t submit request for report generation again, it is showing processing, how can we change the status from backend so that we enable user again to submit a new request, is there something that we can check in Database? 

please find attached screenshot for more details:
![104OPaqJuKutaVMf3RaAcARlZHlesQqSAQ-20230509-072657](https://github.com/Sunbird-Lern/Community/assets/125978181/52185a63-6226-4b22-bf96-4025d2883d6e)
![tIeDZZy7r1SzvpWwBg-wSNsJ-Q-Rn9bPCQ-20230509-072718](https://github.com/Sunbird-Lern/Community/assets/125978181/f47ad403-53a4-41af-aad2-722e834200e4)
![xbQ-0mFl1mSQW764t9mKr7kAPnyeJ8iVpg-20230515-082157](https://github.com/Sunbird-Lern/Community/assets/125978181/b96d4eb9-2ea3-49d5-a38f-61c1fa2d9901)

**Comment by shridhar95** [comment]
Hi Team,

Could you please help on these issue.

Thanks

**Reply by kumarks1122** [reply]
@shridhar95 : Job request gets stored in postgres DB. postgres DB details can be found in the analytics service config.
Table name: `{{env}}_job_request`
DB name: `analytics`
Also, user can submit the request for a batch only once within the same day.

**Reply by kumarks1122** [reply]
cc: @Krishnaj20 @anandvarada @vinukumar-vs

**Reply by shridhar95** [reply]
Thank you so much for your kind response. it helped a lot.

**Reply by AmiableAnil** [reply]
@shridhar95 Please let us know if this the issue is resolved. If not, so please let know, so that we can help you to resolve it further.

**Comment by shridhar95** [comment]
Hi @AmiableAnil  Thanks for your response,
We had worked on same. but the issue is still pending related question set report status is in under processing. 
Could you please help on how to resolve this issue?

Thanks

**Reply by kumarks1122** [reply]
@shridhar95 : Same kind of issue is been discussed in the below thread. Kindly check it.
https://github.com/orgs/Sunbird-Lern/discussions/124
Please let us know if anything is required.

**Reply by shridhar95** [reply]
Hi @kumarks1122  
As per your instruction we did the same and now we are still waiting for user confirmation.
Once we get the confirmation will update and close this issue.

Thanks

**Reply by Jayaprakash8887** [reply, marked as answer]
@shridhar95 Please mark this thread as answered if there are no further queries.

**Comment by shridhar95** [comment]
Hi Team,
Thanks for your kind support, we have followed this steps as suggested and now we are able to get the reports.
